Common/Constants/Constants.cs

```diff
@@ -7,13 +7,18 @@ public class Constants
         public const string CorectEmail = "example33@gmail.com";
         public const string CorectPassword = "1234567";
         public const string ClaimUserId = "UserId";
-        public const int Validity = 1;
+        public const int TokenExpirationInHours = 1;
         public const string PasswordInvalidMessage = "The Password field is not a valid password";
         public const string PasswordRegularExpression = @"^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=.,\-_!])([a-zA-Z0-9 @#$%^&+=*.,\-_!]){8,}$";
-        public const string ErrorMessageUserNotFound = "User does not Exist!";
-        public const string ErrorMessageIncorrectPassword = "Incorrect password entered";
+        public const string ErrorMessageLoginFailed = "Incorrect credentials entered";
+        public const string ErrorMessageIncorrectPassword = "Incorrect credentials entered";
         public const string GoogleResponsePath = "https://www.googleapis.com/oauth2/v1/userinfo?access_token=";
         public const string FacebookResponsePath = "https://graph.facebook.com/v3.2/me?fields=email,first_name,last_name&access_token=";
+        public const string JwtTokenTest = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySWQiOiI0OTciLCJzdWIiOiJzYXNoYXR5bW9zaGNodWswN0BnbWFpbC5jb20iLCJqdGkiOiJiMjA1NTYzZS01ZDZmLTQ3MjktYmZmNC1hNWNlYjRjYjI2YmIiLCJleHAiOjE1NDcxMDg1MDIsImlzcyI6IlRva2VuczpJc3N1ZXIiLCJhdWQiOiJUb2tlbnM6QXVkaWVuY2UifQ.yjE7zzCNUk8aMGkIkkmZk2IFEs-b8CvpT6r-OLLI2GA";
+        public const string ErrorMessageInvalidToken = "Access token is invalid";
+        public const string FacebookValidToken = "EAAMnrYZCCs68BAFLwsiOAXBi95piRosjnvU5BeZCkfxeTzS7lD5PCZAZBxXNogkGAkAw2N3arefWlXq7e6qb2HAKgyXj9Fh8y3szccJZBZAFPCYns6NYN2WjqkBnCLrWr3n7gbc07qspNZB8e61XPBhkUiFyVSFbF3i2W24yrwJ6gZDZD";

```

Comment:
used in test? Cpmmon is not a good place for such consts. Just move to test itself, or if it's used in multiple tests - create new TestConsts class for it.

Reply:
ok, will move to test class